templates/app.html

```diff
@@ -88,7 +88,13 @@ <h5 class="header center col s12 light">Search the database here.</h5>
 					</div>
 				</div>
 				<div class='row'>
-					{{ form.submit(class="col s12 btn btn-large waves-effect green") }}
+          <i class="col s12 btn btn-large waves-effect green waves-input-wrapper" style="
+    padding: 0;
+"><input class="col s12 waves-button-input" id="submit" name="submit" type="submit" value="Get Results" style="
+    height: 100%;
+    padding: 0;
+"></i>
+					<!-- {{ form.submit(class="col s12 btn btn-large waves-effect green") }} -->
```

Comment:
Added a button that is the exact same style as the old one but this one works.

Reply:
The "Get Results" button isn't a written HTML button. It's passed in from the wtforms `SubmitField` class. Styling it would require a class to be made to the CSS file. I'll do that with the style you wrote. Thanks!